client/src/components/Standups/StandupCreate.tsx

```diff
@@ -19,14 +39,7 @@ const StandupCreate = () => {
             <TextInput source="content" fullWidth />
           </SimpleFormIterator>
         </ArrayInput>
-        <SelectInput
-          source="Channels"
-          choices={[
-            { id: "channel 1", name: "Channel 1" },
-            { id: "channel 2", name: "Channel 2" },
-            { id: "channel 3", name: "Channel 3" },
-          ]}
-        />
+        <Dropdown list={channelList} />
```

Comment:
As we are not writing test cases, do we need a component ? can we do
```suggestion
        <SelectInput choices={channelList} />
```